doc/topics/releases/2017.7.0.rst

```diff
@@ -26,6 +21,16 @@ Salt will no longer support Python 2.6. We will provide python2.7 packages on ou
 
 .. _repo: https://repo.saltstack.com/
 
+============
+Known Issues
+============
+The following salt-cloud drivers have known issues running with Python 3. These drivers will not work with Python 3, and Python 2.7 should be used instead:
+
+- Joyent
```

Comment:
I think this is more appropriate as a list because we are listing off the cloud drivers that are affected. There isn't any other information about Joyent in the docs to add to the subsection